### PR TITLE
add some troubleshooting steps

### PR DESCRIPTION
**Pull request by Andtalath**

### CODE REVIEW

File: content/for-developers/tutorials/inner-loop/expose-metrics/expose-metrics.md [hunk 1]
```diff
@@ -57,3 +57,28 @@ Now, let's add the `ServiceMonitor` for our `stakater-nordmart-review-api` appli
...
+
+```sh
+oc get pods -n namespace
+oc debug pod/podname-from-above
+curl localhost:port/path
+```
+
+- Verify everything in spec in the ServiceMonitor.
+    - Verify that endpoints port matches with the Service spec.ports.name .
+    - Verify that namespaceSelector matchnames matches with the Service metadata.namespace .
```
Comment (rasheedamir) [inline content/for-developers/tutorials/inner-loop/expose-metrics/expose-metrics.md:75]: @Andtalath can we give example of matches? Both should have same number of selectors or can be different on any side? Any examples we can add for newbies? In what cases it won't work?

File: content/for-developers/tutorials/inner-loop/expose-metrics/expose-metrics.md [hunk 2]
```diff
@@ -57,3 +57,28 @@ Now, let's add the `ServiceMonitor` for our `stakater-nordmart-review-api` appli
...
+    - Verify that selector matchlabels matches with the Service metadata.labels .
+
+- Make sure that the service monitor selector and service monitor namespace selector labels have been applied to the service monitor and the namespace respectively where the service monitor is present. You can check these selectors in Search -> Resources -> Prometheus and then check the yaml manifest of Prometheus CR.
+
+- If nothing else works then start with port-forwarding the Prometheus pod to port: 9090 and navigate to Prometheus UI with http. Check in the service discovery that service monitor is picked up, also check the targets that Prometheus is able to scrape the metrics. If the target is visible, but metrics are not scraped then the error should be visible in targets.
```
Comment (rasheedamir) [inline content/for-developers/tutorials/inner-loop/expose-metrics/expose-metrics.md:80]: @Andtalath can we add some screenshots here to make it easy?

Comment (Andtalath) [reply content/for-developers/tutorials/inner-loop/expose-metrics/expose-metrics.md:80]: Not really sure what screenshots are useful without also introducing confusing examples as well.

File: content/for-developers/tutorials/inner-loop/expose-metrics/expose-metrics.md (hunk 1 above)
Comment (Andtalath) [reply content/for-developers/tutorials/inner-loop/expose-metrics/expose-metrics.md:75]: We could. Best way is probably a couple of scrubbed manifests.
However, the issue then becomes the level of specifity to document.

Comment (Andtalath) [reply content/for-developers/tutorials/inner-loop/expose-metrics/expose-metrics.md]: Outside of conversation we agreed that providing examples is somewhat superflous in this context.

File: content/for-developers/tutorials/inner-loop/expose-metrics/expose-metrics.md (hunk 2 above)
Comment (Andtalath) [reply content/for-developers/tutorials/inner-loop/expose-metrics/expose-metrics.md:80]: We agreed outside of her that screenshots will more likely confuse users with too specific labels.

**Review by rasheedamir — APPROVED** [review]

**Pull request merged**